### PR TITLE
Introduce simple API retry

### PR DESCRIPTION
**Pull request by polarker**

### CODE REVIEW

File: packages/web3/src/api/utils.ts [hunk 1]
```diff
@@ -16,10 +16,24 @@ You should have received a copy of the GNU Lesser General Public License
...
   } else {
     return response.data
   }
 }
+
+export async function retryFetch(...fetchParams: Parameters<typeof fetch>): ReturnType<typeof fetch> {
+  const retry = async (retryNum: number): ReturnType<typeof fetch> => {
+    const response = await fetch(...fetchParams)
+    if (response.status === 429 && retryNum < 5) {
+      return await retry(retryNum + 1)
```
Comment (Lbqds) [inline packages/web3/src/api/utils.ts:33]: Do we need to wait for some time before starting the next request?

**Review by Lbqds — APPROVED** [review]
LGTM, only one question.

File: packages/web3/src/api/utils.ts (hunk 1 above)
Comment (polarker) [reply packages/web3/src/api/utils.ts]: Great catch. I had that in mind but forgot it somehow 😓

**Review by Lbqds — APPROVED** [review]

**Review by h0ngcha0 — APPROVED** [review]
LGTM :+1:

**Pull request merged**